packages/kit/src/views/Market/MarketHomeV2/components/MarketTokenList/hooks/useMarketTokenList.ts

```
@@ -0,0 +1,100 @@
+
+// 将 API 返回的数据转换为组件使用的格式
+function transformApiDataToComponentData(
+  apiData: IMarketTokenListItem[],
+): IMarketToken[] {
+  return apiData.map((item, index) => ({
+    id: item.address || `${index}`,
+    name: item.name,
+    symbol: item.symbol,
+    address: item.address,
+    price: parseFloat(item.price || '0'),
+    change24h: parseFloat(item.priceChange24hPercent || '0'),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)